lib/mollie/customer/mandate.rb

```diff
@@ -3,28 +3,33 @@ class Customer
     class Mandate < Base
       STATUS_VALID   = "valid"
       STATUS_INVALID = "invalid"
+      STATUS_PENDING = "pending"
 
       attr_accessor :id,
                     :status,
                     :method,
                     :customer_id,
                     :details,
-                    :created_datetime,
-                    :mandate_reference
-
+                    :mandate_reference,
+                    :signature_date,
+                    :created_at
```

Comment:
Missing the `links` to `_links` alias and the helper method to directly access the `customer` object within `_links.customer`.